Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs

```diff
@@ -194,10 +194,18 @@ private async Task<List<DocumentToConvert>> RefreshElementsIdsOnSender(SenderMod
       .Where(el => el is not null)
       .ToList();
 
-    // TODO: currently below part suits only for selection, need more work to see how align with other filters
-    List<DocumentToConvert> elementsByTransform = [new(null, activeUIDoc.Document, elementsOnMainModel)];
+    // TODO: [CNX-1377] currently below part suits only for selection, need more work to see how align with other filters
+    List<DocumentToConvert> documentElementContexts = [new(null, activeUIDoc.Document, elementsOnMainModel)];
 
     var linkedModels = elementsOnMainModel.Where(el => el is RevitLinkInstance).Cast<RevitLinkInstance>().ToList();
+
+    // are there linked models selected but the setting is deactivated? then we should notify the user
+    if (linkedModels.Count > 0 && !_toSpeckleSettingsManager.GetLinkedModelsSetting(modelCard))
+    {
+      // TODO: log as warning possible in this context?
+      throw new SpeckleException("Linked models detected but UI setting not activated. ");
```

Comment:
I wouldn't bork whole send operation for this, we could handle it on root object builder and error out as report error

Reply:
Yup yup totally agree. Let's sync on this